File: ItemService/src/main/java/main/helpers/AddBookResult.java
package main.helpers;

public class AddBookResult {
	
	public Long bookId;
	
	public String userId;
	
	public AddBookResult() {}
	
	public AddBookResult(Long bookId, String userId) {
		this.bookId=bookId;
		this.userId=userId;
	}

}

File: UserService/UserService/Helpers/AddBookResult.cs
using Microsoft.AspNetCore.Http;
using Microsoft.AspNetCore.Mvc;

namespace UserService.Helpers
{
    public class AddBookResult
    {
        public long bookId { get; set; }
        public string userId { get; set; }

        public AddBookResult() { }

        public AddBookResult(long ItemId, string userId)
        {
            this.bookId = ItemId;
            this.userId = userId;
        }
    }
}
